components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue

```diff
@@ -1,21 +1,24 @@
 <template>
   <div class="gallery-activity-events-wrapper is-flex is-flex-direction-column">
-    <div class="events p-5 is-flex is-flex-direction-column">
+    <div class="events px-5 py-4 is-flex is-flex-direction-column">
       <div class="events-filter is-flex is-flex-wrap-wrap">
-        <div class="events-checkbox" @click="checkAll">
+        <div
+          class="events-checkbox is-flex is-align-items-center"
```

Comment:
```suggestion
class="is-clickable is-capitalized is-flex is-align-items-center"
```

same goes for other uses of events-checkbox class